src/email_requests/views.py

```diff
@@ -35,14 +36,14 @@ def post(self, request, *args, **kwargs):
         #TODO: Use SendGrid Templates
         content_string = '\n\n'.join([
             "You have a new request from " + mentee_name,
-            "Their email is: " + preferred_mentee_email + " and their phone number is "+ phone_num,
+            "Their email is: " + preferred_mentee_email + #" and their phone number is "+ phone_num,
```

Comment:
with the comment, this line fucks up the formattjng

Reply:
The email stuff is all temporary I'll reformat the email entirely tonight in a different pull request. Formatting was already super bad.